File: C/1074.c
#include <stdio.h>
 
int main() {
 
    int n;
    scanf("%d", &n);
    for(int i = 1; i <= n;i++){
        int num;
        scanf("%d", &num);
        if(num == 0){
            printf("NULL\n");
        }else if(num%2){
            if(num > 0){
                printf("ODD POSITIVE\n");
            }else{
                printf("ODD NEGATIVE\n");
            }
        }else{
            if(num > 0){
                printf("EVEN POSITIVE\n");
            }else{
                printf("EVEN NEGATIVE\n");
            }
        }
    }
}

File: Python/1074.py
n = int(input())
lista = []

for i in range(n):
    num = int(input())
    lista.append(num)
for x in lista:
    if x > 0 and x % 2 == 0:
        print('EVEN POSITIVE')
    elif x < 0 and x % 2 == 0:
        print('EVEN NEGATIVE')
    elif x > 0 and x % 2 != 0:
        print('ODD POSITIVE')
    elif x < 0 and x % 2 != 0:
        print('ODD NEGATIVE')
    else:
        print('NULL')